src/panels/developer-tools/logs/system-log-card.ts

```diff
@@ -61,25 +57,30 @@ export class SystemLogCard extends LitElement {
                       </div>
                     `
                   : this._items.map(
-                      (item) => html`
+                      (item, idx) => html`
                         <paper-item @click=${this._openLog} .logItem=${item}>
                           <paper-item-body two-line>
                             <div class="row">
                               ${item.message}
                             </div>
                             <div secondary>
-                              ${formatLogTime(
+                              ${formatSystemLogTime(
                                 item.timestamp,
                                 this.hass!.language
                               )}
+                              ${integrations[idx]
+                                ? html`
+                                    ${integrations[idx]}:
+                                  `
+                                : ""}
                               ${item.source} (${item.level})
```

Comment:
Should we only show the source if not an integration and move the source to the details dialog?

Reply:
I'm so used to using `git commit -am` now because of pre-commit that I forgot to add the new file! fixed.

Reply:
I'm so used to using `git commit -am` now because of pre-commit that I forgot to add the new file! fixed.